### ISSUE TITLE
PSČ a kód pojišťovny by měly být číselné hodnoty

### ISSUE THREAD

**Issue by adelpopelkova** [frenzymadness/donors_registry#188]
Při testování jsem narazila na to, že i toto je validní řádek importu:
```
0123456789;Jméno;Příjmení;Adresa 123/45;Město;PSČ;Pojišťovna;1
```
Co mě na tom zarazilo je, že PSČ a kód pojišťovny nemusí být číslo a import se úspěšně povede.

**Comment by encukou** [comment]
Jen podotknu že podobné identifikátory nejsou čísla. Když něco nedává smysl dělit, sečítat a zaokrouhlovat, když to nedává smysl zapsat v jiné soustavě, když v tom můžou hrát roli nečíselné znaky a mezery na začátku (jako `+420 `/`00420` u telefonního „čísla“), je to lepší ukládat jako řetězec.
A ověřovat třeba regulárním výrazem, když je to potřeba.

**Comment by frenzymadness** [comment]
Ano, dokonce ve starém systému byl bug, kdy se rodné číslo ukládalo jako číslo a proto se zahazovaly úvodní nuly u lidí narozených po roce 2000.
PSČ můžeme zkusit validovat podle obsahu aktuální databáze nebo možných variant u našich sousedů - museli bychom zkontrolovat, jak to funguje na Slovensku a v Polsku.

**Comment by frenzymadness** [comment]
U validace v tomto projektu je obecně problém. Co se má stát s daty, které jsou nevalidní během importu? Pokud je PSČ nesmyslné, jsem před odesláním dopisu schopen jej podle adresy opravit. Když se mi to ale ukáže u 200 lidí, které si importuji i přes to, že s nimi třeba už nikdy nebudu pracovat, tak tam bych to prostě neřešil. Jakože z tisíců importovaných záznamů pak může reálný problém vzniknout jen u jednotech z nich.
Mohlo by tedy dávat smysl validovat PSČ ve chvíli, kdy se připravujeme na udělování medailí.
